src/entities/common-interfaces.ts

```diff
@@ -46,4 +46,17 @@ export interface AbleToExportAndImportEuring {
   importEURING(code: string): any;
 }
 
+export enum SortingDirection {
```

Comment:
we seem to be contaminating the space of this namespace, which was originally intended for models. 

I think that the service itself can export the interfaces for these methods in addition to the methods.